DataConnectors/CEF/cef_AMA_troubleshoot.py

```diff
@@ -421,11 +421,19 @@ def verify_Syslog_daemon_listening(self):
         command_to_run = "sudo netstat -lnpv | grep " + self.SYSLOG_DAEMON
         result_keywords_array = [self.SYSLOG_DAEMON, "LISTEN", ":514 "]
         command_object = BasicCommand(command_name, command_to_run, result_keywords_array)
-        command_object.run_full_test()
-        if command_object.is_successful == "Warn":
-            command_object.print_warning(command_object.command_result_err)
-        elif not command_object.is_successful:
-            command_object.print_warning(self.Syslog_daemon_not_listening_warning)
+        command_object.run_command()
+        is_command_successful = command_object.is_command_successful()
+        if is_command_successful is True:
+            command_object.print_result_to_prompt()
+            command_object.log_result_to_file()
+        else:
+            result_keywords_array = [self.SYSLOG_DAEMON, "LISTEN", ":6514 "]
+            command_object = BasicCommand(command_name, command_to_run, result_keywords_array)
+            command_object.run_full_test()
+            if command_object.is_successful == "Warn":
```

Comment:
Perhaps worth running this check at the top (line 426), because if the command doesn't exist (hence the warning), then it's not worth running this check again.